### PR TITLE
Second batch of GKE alerts

### PR DESCRIPTION
**Pull request by BinaryFissionGames**
This PR contains the second batch of alerting policies for GKE.

The alerting policies included here are for the following technologies:
- Apache Airflow Scheduler
- Argo Server
- Consul
- etcd
- HAProxy
- Istio Proxy
- Jenkins
- Kibana
- Kube State Metrics
- NGINX Ingress Controller
- Node Exporter
- ScyllaDB
- Velero

### CODE REVIEW

**Comment by chanseokoh** [comment]
I've done syntax reviews on all the `.json` files. They look good to me.

@adeychman I'll leave the approval up to you.

File: alerts/etcd-gke/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# etcd Alerts for GKE
+
+## Long fsync duration alert
+When the WAL (Write-Ahead Logging) fsync time is too long, performance is degrading, often due to disk issues. This could cause cluster instability, or high latency for client requests.
```
Comment (adeychman) [inline alerts/etcd-gke/README.md:4]: please remove the comma after "instability"

**Comment by adeychman** [comment]
@BinaryFissionGames One minor comment, LGTM! Thank you

File: alerts/etcd-gke/README.md (hunk 1 above)
Comment (BinaryFissionGames) [reply alerts/etcd-gke/README.md]: Removed the comma from README, metadata, and json.

**Pull request merged**